layouts/partials/widgets/social.html

```diff
@@ -70,7 +70,23 @@ <h4 class="widget-social__title widget__title">{{ T "social_title" }}</h4>
 		<div class="widget-social__item widget__item">
 			<a class="widget-social__link widget__link btn" title="Email" href="mailto:{{ . }}">
 				{{ partial "svg/email.svg" (dict "class" "widget-social__link-icon") }}
-				<span>{{ . }}</span>
+				<span>Email</span>
```

Comment:
I think, it makes no sense. Bots just will find `a@href="mailto:"`.

Reply:
Then this is perhaps a style preference rather than a security issue. You can ignore this change if you would like to.